jetty-servlet/src/main/java/org/eclipse/jetty/servlet/ServletContextHandler.java

```diff
@@ -1382,6 +1383,38 @@ public void setSessionTrackingModes(Set<SessionTrackingMode> sessionTrackingMode
                 _sessionHandler.setSessionTrackingModes(sessionTrackingModes);
         }
 
+        @Override
+        public int getSessionTimeout()
+        {
+            if (!isStarting())
+                throw new IllegalStateException();
+            if (!_enabled)
+                throw new UnsupportedOperationException();
+
+            int timeout = -1;
+            if (_sessionHandler != null)
+            {
+                int maxInactive = _sessionHandler.getMaxInactiveInterval();
+                timeout = maxInactive / 60;
```

Comment:
Division by 60 here, but missing multiplication in the setter?
They should be in the same class.

Reply:
OK, I've added getMaxInactiveIntervalMins() on SessionHandler, so all mins<->seconds happens entirely in SessionHandler.